react/react1/week1/to-do-app/src/App.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
```

Comment:
I believe you forgot to import ToDo here.
```suggestion
    <div className="App"> <ToDo />
```
by the way, take care with over nesting of components.

Reply:
yes actually  I have exported the components to index.js file from page.jsx

Reply:
But yes I will take a care thanks😊